2021/21xxx/CVE-2021-21562.json

```diff
@@ -1,17 +1,70 @@
 {
```

Comment:
'ttps://www.dell.com/support/kbdoc/000188148' does not match '^(ftp|http)s?://\\S+$'